core/src/js/utilities/events.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Create an event with the specified name in a browser-agnostic way.
+ * @param {string} eventName - the name of the event
+ * @return {Event} - instance of event which can be dispatched / listened for
+ */
+export const createEvent = (eventName = '') => {
```

Comment:
Do you need the default string? I'd vote no as it is not useful to have an empty string passed in.